networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -0,0 +1,36 @@
+// AWS Load Balancer Operator Release Notes
+:_content-type: ASSEMBLY
+[id="aws-load-balancer-operator-release-notes"]
+= AWS Load Balancer Operator release notes
+:context: aws-load-balancer-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The AWS Load Balancer (ALB) Operator deploys and manages an instance of the `aws-load-balancer-controller` resource.
+
+These release notes track the development of the AWS Load Balancer Operator in {product-title}.
+
+For an overview of the AWS Load Balancer Operator, see xref:../../networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc#aws-load-balancer-operator[AWS Load Balancer Operator in {product-title}].
+
+[id="aws-load-balancer-operator-release-notes-1.0.0"]
+== AWS Load Balancer Operator 1.0.0
+
+The following advisory is available for the AWS Load Balancer Operator version 1.0.0:
+
+// TODO: Replace "????" with the actual advisory once created.
+* link:https://access.redhat.com/errata/RHEA-2023:????[RHEA-2023:???? Release of AWS Load Balancer Operator on OperatorHub Enhancement Advisory Update]
+
+[id="aws-load-balancer-operator-1.0.0-bug-fixes"]
+=== Bug fixes
+
+* Previously, the controller provisioned by the AWS Load Balancer Operator did not properly use the configuration for the cluster-wide proxy.
```

Comment:
I don't know whether we want to mention the jira ticket or not but here its number: OCPBUGS-5295.

Reply:
Aha, good point. I was thinking about that while writing this, but apparently forgot to include the information. I'll update this in a little while with a link to the issue. Thank you! :slightly_smiling_face:

Reply:
@alebedev87 Related, is there anything that we should say about OCPBUGS-4052? The procedure documented there already exists in the docs set, so I'm unsure if we need to say anything in particular about it. This can be a small release note that points to that documentation if we want to mention it.

Reply:
I think we can mention both of these bugs, under the umbrella description you made.